nodewatcher/modules/analysis/channel_allocation/allocation_algorithms.py

```diff
@@ -0,0 +1,327 @@
+import networkx as nx
+
+from nodewatcher.modules.analysis.channel_allocation import channel_lookup
+from nodewatcher.modules.analysis.channel_allocation import signal_processing
+
+
+def meta_algorithm(graph, known_nodes):
+    """
+    Ensures that specific algorithms are called with appropriate data structures and chooses the algorithm.
+
+    :param graph: A graph datastructure which contains both friendly and unknown nodes.
+    :param known_nodes: An array of BSSIDs monitored by nodewatcher.
+    :return: A dictionary with two k-v pairs, one for each frequency spectrum. Keys are "2.4GHz" and "5GHz".
```

Comment:
Mostly because of the consistency with our data collection. We use the same network interface for both ranges on 5GHz, and a different one for 2.4GHz. How should we clarify this?